ChangeLog

```diff
@@ -1,5 +1,12 @@
 mbed TLS ChangeLog (Sorted per branch, date)
 
+= mbed TLS x.x.x branch released xxxx-xx-xx
+
+Bugfix
+   * Add ecc extensions only if an ecc based ciphersuite is used.
```

Comment:
I think ChangeLog entries should provide more context. For example here, "In TLS, add ECC extensions to the ClientHello and ServerHello messages only if". (And ECC should be capitalized while at it.)

Also, "Affects interop" is ambiguous and not a complete sentence. How about: "This improves compliance to RFC 4492, and as a result, solves interoperability issues with BouncyCastle."?